geode-core/src/test/java/org/apache/geode/management/internal/cli/domain/RegionDescriptionJUnitTest.java

```diff
@@ -0,0 +1,312 @@
+  }
+
+  @Test
+  public void simpleDescribeRegionCommandConsumesRegionDescriptionAsExpected() {
+    RegionDescriptionPerMember mockDescPerMember = getMockRegionDescriptionPerMember_A();
+    RegionDescription regionDescription = new RegionDescription();
+    regionDescription.add(mockDescPerMember);
+    CommandResult commandResult =
+        (CommandResult) command.buildDescriptionResult(regionName, regionDescription);
+    assertThat(commandResult.getStatus()).isEqualTo(Status.OK);
+    // We can't assert that the "Non-Default Attributes Specific To The Hosting Members" header is
+    // missing because Gfsh currently obfuscates portions of the underlying json that have empty
+    // "content" fields. We'll have to dig into the JSON to verify.
+    JSONObject shared = getSharedAttributedJson(commandResult);
+    JSONObject unique = getMemberSpecificAttributeJson(commandResult);
+    assertThat(shared.toString()).contains(evictionKeyShared, partKeyShared, regKeyShared,
+        evictionValueShared, partValueShared, regValueShared, evictionKeyA, partKeyA, regKeyA,
+        evictionValueA, partValueA, regValueA);
+    assertThat(unique.toString()).isEqualTo("{}");
+  }
+
+  @Test
+  public void compoundDescribeRegionCommandConsumesRegionDescriptionAsExpected() {
+    RegionDescriptionPerMember mockDescPerMemberA = getMockRegionDescriptionPerMember_A();
+    RegionDescriptionPerMember mockDescPerMemberB = getMockRegionDescriptionPerMember_B();
+    RegionDescription regionDescription = new RegionDescription();
+    regionDescription.add(mockDescPerMemberA);
+    regionDescription.add(mockDescPerMemberB);
+    CommandResult commandResult =
+        (CommandResult) command.buildDescriptionResult(regionName, regionDescription);
+    assertThat(commandResult.getStatus()).isEqualTo(Status.OK);
+    // We can't assert that the "Non-Default Attributes Specific To The Hosting Members" header is
+    // missing because Gfsh currently obfuscates portions of the underlying json that have empty
+    // "content" fields. We'll have to dig into the JSON to verify.
+    JSONObject shared = getSharedAttributedJson(commandResult);
+    JSONObject unique = getMemberSpecificAttributeJson(commandResult);
+    assertThat(shared.toString()).contains(evictionKeyShared, partKeyShared, regKeyShared,
+        evictionValueShared, partValueShared, regValueShared);
+    assertThat(unique.toString()).contains(evictionKeyA, evictionValueA, partKeyA, partValueA,
+        regKeyA, regValueA, evictionKeyB, evictionValueB, partKeyB, partValueB, regKeyB, regValueB);
+  }
+
+  private JSONObject getSharedAttributedJson(CommandResult commandResult) {
+    return commandResult.getContent().getInternalJsonObject().getJSONObject("__sections__-0")
+        .getJSONObject("__sections__-0").getJSONObject("__tables__-0").getJSONObject("content");
+  }
+
+  private JSONObject getMemberSpecificAttributeJson(CommandResult commandResult) {
```

Comment:
use commandResult.getTableContent

Reply:
I will add some logic to the `CommandResult` class to go digging into the structure, since it seems to belong to that more than this class.  Unfortunately, I can't just use `getTableContent` because the structure here is significantly different than in other commands, notably that this commandResult has two `__section__-x` levels before the tables.